documentation/español/implementacion.MD

```diff
@@ -17,93 +17,167 @@
-    - [Eliminación de los mensajes redundantes usando la Multicast Route Message Set](#eliminaci%c3%b3n-de-los-mensajes-redundantes-usando-la-multicast-route-message-set)
-    - [Creación de mensajes RREQ](#creaci%c3%b3n-de-mensajes-rreq)
-    - [Recepción de mensajes RREQ](#recepci%c3%b3n-de-mensajes-rreq)
-    - [Reenvío de mensajes RREQ](#reenv%c3%ado-de-mensajes-rreq)
-    - [Recepción de mensajes RREP](#recepci%c3%b3n-de-mensajes-rrep)
-    - [Reenvío de mensajes RREP](#reenv%c3%ado-de-mensajes-rrep)
-    - [Generación de mensajes RREP_Ack Request](#generaci%c3%b3n-de-mensajes-rrepack-request)
-    - [Recepción de mensajes RREP_Ack](#recepci%c3%b3n-de-mensajes-rrepack)
-    - [Generación de RREP_Ack Response](#generaci%c3%b3n-de-rrepack-response)
-- [Formato de paquetes para redes MANET](#formato-de-paquetes-para-redes-manet)
-  - [Paquetes](#paquetes)
-    - [pkt-header](#pkt-header)
-  - [Mensajes](#mensajes-1)
-    - [msg-header](#msg-header)
-    - [msg-header-length](#msg-header-length)
-  - [oonf_api](#oonfapi)
-- [Modelado de AODVv2](#modelado-de-aodvv2)
-  - [Estructuras de datos](#estructuras-de-datos)
-  - [Mensajes](#mensajes-2)
-  - [Estado Inicial](#estado-inicial)
-  - [Acciones del protocolo](#acciones-del-protocolo)
-  - [**Modelo para crear un Route Request**](#modelo-para-crear-un-route-request)
-  - [**Modelo para recibir un Route Request**](#modelo-para-recibir-un-route-request)
-  - [**Modelo para recibir un Route Reply**](#modelo-para-recibir-un-route-reply)
-  - [**Modelo para recibir un Route Error**](#modelo-para-recibir-un-route-error)
-  - [**Modelo para remover un link**](#modelo-para-remover-un-link)
-- [Operaciones generales del protocolo AODVv2](#operaciones-generales-del-protocolo-aodvv2)
-  - [Operaciones de ruteo](#operaciones-de-ruteo)
-  - [Process_Routing_Info](#processroutinginfo)
-  - [Create_Route_Table_Entry](#createroutetableentry)
-  - [update_route_table_entry](#updateroutetableentry)
-  - [fetch_route_table_entry](#fetchroutetableentry)
-  - [loop_free](#loopfree)
-  - [check_route_state](#checkroutestate)
-- [Tabla de operaciones para los mensajes de ruta multicast](#tabla-de-operaciones-para-los-mensajes-de-ruta-multicast)
-  - [fetch_rte_msg_table_entry](#fetchrtemsgtableentry)
-  - [update_rte_msg_table](#updatertemsgtable)
-- [Generación y procesamiento de mensajes](#generaci%c3%b3n-y-procesamiento-de-mensajes)
-  - [build_rfc_5444_message_header](#buildrfc5444messageheader)
-  - [receive_RREQ](#receiverreq)
-  - [generate_rreq](#generaterreq)
-  - [regenerate_RREQ](#regeneraterreq)
-  - [generate_rrep](#generaterrep)
-  - [receive_rrep](#receiverrep)
-  - [regenerate_rrep](#regeneraterrep)
-
-# Introducción
-El objetivo de este proyecto es la implementación del protocolo **AODVv2**, el cual se enmarca en uno de mayor envergadura que comprende las siguientes fases.
+<h1> Tabla de contenido</h1>
```

Comment:
```suggestion
# Tabla de contenido
```

This can be the same, markdown converts one `#` to `<h1>`

Reply:
He usado h1 con el fin de quitar el titulo "Tabla de contenido" de la tabla de contenido